CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [2.1.2] - 2020-11-26
```

Comment:
I can handle this myself after merging, but for the future - the level 2 headings in [the changelog format used here](https://keepachangelog.com) is only added when we do an actual release - which means bumping the version. The date is the date of the release, not the date of the change. As long as you don't make a release, you shouldn't add that heading and instead log your changes under the `## Unreleased` heading.

Reply:
Oh okay. Wasnt really familiar with the process. Thanks for adding this fix!